Standards/CCSMB-3.md

```diff
@@ -0,0 +1,169 @@
+# CCSMB-3 bimg File Format
+
+*Author: SkyTheCodeMaster*
+
+*Contributors: MasonGulu, spr2-dev*
+
+*Version: 1.0.0*
+
+*Last Revised: 2022-07-29*
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+bimg (abbreviation of "blit image") is a versatile image format for the ComputerCraft Minecraft mod.
+This specification aims to provide a standardized image format that can be easily implemented and used throughout the entire ComputerCraft ecosystem.
+
+# File Structure
```

Comment:
A single hash is for the title of the document afaik
Markdown should also have a newline before and after headings, codeblocks